interfaces/Portalicious/src/app/pages/project-registrations/components/change-status-reason/change-status-reason.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="space-y-4">
```

Comment:
```suggestion
<div class="my-4">
```

I think maybe you're looking for margin, not space. Space is for space in between children, but there is only one child here.